app/Http/Controllers/CaseStudiesController.php

```diff
@@ -24,7 +24,10 @@ public function get(Factory $factory, string $slug): View
 
         $view = preg_replace('/\.blade\.php$/', "", $caseStudy->template);
 
+        $locale = app()->getLocale();
+
         return $factory->make("case-studies/$view")
```

Comment:
Don't you want to check if this file exists?

Reply:
Added